openhcl/underhill_core/src/worker.rs

```diff
@@ -1442,6 +1450,19 @@ async fn new_underhill_vm(
     .await
     .context("failed to initialize memory")?;
 
+    // Devices in isolated VMs default to accessing only shared memory, since
+    // that is what the guest expects--it will double buffer memory to be DMAed
+    // through a shared memory pool.
+    //
+    // For non-isolated VMs, there is no shared/private distinction, so devices
```

Comment:
err, this comment doesn't include if we _hide_ isolation, that we treat it as a non-isolated VM that can only access confidential memory?